### PR TITLE
Port files over to `pyproject.toml`

### LINKED ISSUE #88: Remove `setup.cfg` in favour of `pyproject.toml`

### PR DESCRIPTION
**Pull request by paddyroddy**
Fixes #88. Needs testing.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,17 +1,70 @@
...
+    "Framework :: napari",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: BSD License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python",
+    "Topic :: Software Development :: Testing",
```
Comment (p-j-smith) [inline pyproject.toml:24]: it's not really a testing framework - [napari has](https://github.com/napari/napari/blob/main/setup.cfg#L29):
```
Topic :: Scientific/Engineering :: Visualization
``` 
which might be a better fit

**Review by p-j-smith — CHANGES_REQUESTED** [review]
nice! just a couple of questions about the project classifiers

File: pyproject.toml [hunk 2]
```diff
@@ -1,17 +1,70 @@
...
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Framework :: napari",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: BSD License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
```
[the remark this answers is not in the document]
Comment (paddyroddy) [reply pyproject.toml:22]: Yes, it had indeed. I've now changed it to `sort_inline_arrays = false` so this won't happen now.

File: pyproject.toml (hunk 1 above)
Comment (paddyroddy) [reply pyproject.toml:24]: Sounds sensible

**Review by p-j-smith — APPROVED** [review]

**Pull request merged**